New functions for Catalan and Euler-Mascheroni mathematical constants.

carandraug opened the pull request:
Adds two new functions for the symbolic package. I also have a python script to generate them automatically but in the end I thought that although quite simple, the script would be overkill.

cbm755 commented on inst/catalan.m:
```diff
@@ -0,0 +1,37 @@
+function g = catalan ()
+  g = python_cmd ('return sympy.S.Catalan,');
+end
+
```
trailing white space.

(Usually I want tests although I'm soon going to support doctests so maybe its overkill...)

cbm755 commented on inst/eulergamma.m:
```diff
@@ -0,0 +1,37 @@
+%% Return Euler-Mascheroni constant.
+%%
+%% @example
+%% vpa (catalan ())
```
eulergamma

(and with every such error my belief in doctests grows stronger!)

cbm755 commented on inst/eulergamma.m:
```diff
@@ -0,0 +1,37 @@
+function g = eulergamma ()
+  g = python_cmd ('return sympy.S.EulerGamma,');
+end
+
```
trailing white space

cbm755 commented:
+1 after those changes.

carandraug commented:
About the typo on documentation [here's the script](https://bpaste.net/show/16011b9109db) mentioned on the pull request (with this problem fixed) I guess it could be adjusted to fit with the other script you have on src/ but I didn't want to mess with your build system. Also, Matlab seems to only have this two functions, so unless you want to have the other constants in symbolic, there's even less reason for it.

I have also added two simple tests. I only started playing with the symbolic package this weekend (I was having fun implementing `psi` for Octave core) so I'm unsure what would be the best way.

I have also removed the last empty line as requested.

cbm755 commented:
Awesome thanks for this.  those tests should be fine.  This is in.

>  but I didn't want to mess with your build system.

That was very polite ;-)  But yes it is horrid.

I used to do code generation at install time, but now I've just checked the results in: slightly lowers barrier for new contributions (I hope).  

On the topic of build systems, is https://github.com/cbm755/octsympy/issues/196 an easy fix for you?
